tgui/packages/tgui/interfaces/RoboticsControlConsole.js

```diff
@@ -13,11 +13,11 @@ import { Window } from '../layouts';
 
 export const RoboticsControlConsole = (props, context) => {
   const { act, data } = useBackend(context);
-  const { can_hack, safety, show_detonate_all, cyborgs = [] } = data;
+  const { can_hack, safety, show_lock_all, cyborgs = [] } = data;
   return (
     <Window resizable>
       <Window.Content scrollable>
-        {!!show_detonate_all && (
+        {!!show_lock_all && (
           <Section title="Emergency Self Destruct">
```

Comment:
Should be an emergency lock all title, rather than emergency self destruct button

Reply:
I think this might be on an older commit? It's `show_lock_all` for me. Wack.

Reply:
Wait im dumb.

Reply:
I believe they mean your Section Title should be changed, as it still says Emergency Self Destruct